$NetBSD: patch-ab,v 1.21 2010/05/21 10:05:20 sbd Exp $

Split the install target in installcups & installconf.
(installcup is everything but the config files.)

--- Makefile.orig	2010-01-18 22:10:29.000000000 +0000
+++ Makefile
@@ -30,7 +30,7 @@ include Makedefs
 DIRS	=	cups filter backend berkeley cgi-bin driver locale man monitor \
 		notifier ppdc scheduler systemv test \
 		$(PHPDIR) \
-		conf data doc examples $(FONTS) templates
+		data doc examples $(FONTS) templates
 
 
 #
@@ -162,7 +162,8 @@ ctags:
 # Install everything...
 #
 
-install:	install-data install-headers install-libs install-exec
+install:	install-data install-conf install-headers install-libs install-exec
+installcups:	install-data install-headers install-libs install-exec
 
 
 #
@@ -177,6 +178,11 @@ install-data:
 	echo Installing cups-config script...
 	$(INSTALL_DIR) -m 755 $(BINDIR)
 	$(INSTALL_SCRIPT) cups-config $(BINDIR)/cups-config
+
+install-conf:
+	(cd scheduler; $(MAKE) $(MFLAGS) install-dirs) || exit 1
+	echo Installing in conf...
+	(cd conf; $(MAKE) $(MFLAGS) install)
 	if test "x$(INITDIR)" != x; then \
 		echo Installing init scripts...; \
 		$(INSTALL_DIR) -m 755 $(BUILDROOT)$(INITDIR)/init.d; \
